contents/blog/product-engineer-vs-software-engineer.md

```diff
@@ -14,71 +15,77 @@ crosspost:
   - Product engineers
 ---
 
+
 Software companies were once dominated by two types of roles: product managers and software engineers, but this is changing. A new role is emerging: the [product engineer](/blog/what-is-a-product-engineer).
 
-Want proof? Look at this graph:
+Want proof? [Look at this graph](https://www.youtube.com/watch?v=oMRgdpXLElE):
 
 ![product-engineer-trends](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/blog/product-engineer-trend.png)
 
-Online searches for product engineers have grown by 80% since 2021 as startups like [ourselves](https://newsletter.posthog.com/p/beyond-the-10x-engineer), [incident.io](http://incident.io/), and [Ghost](https://ghost.org/), prioritize engineers who are customer obsessed, talk to them directly, and comfortable working autonomously. 
+Online searches for product engineers have grown by 80% since 2021 as startups like [ourselves](https://newsletter.posthog.com/p/beyond-the-10x-engineer), [incident.io](http://incident.io/), and [Ghost](https://ghost.org/), prioritize engineers who are customer-obsessed, able to talk to users directly, and comfortable working autonomously. 
```

Comment:
```suggestion
Online searches for product engineers have grown by 80% since 2021 as startups like [ourselves](https://newsletter.posthog.com/p/beyond-the-10x-engineer), [incident.io](http://incident.io/), and [Ghost](https://ghost.org/), prioritize engineers who are customer-obsessed, talk to users directly, and comfortable working autonomously. 
```